solidity/test/BondedECDSAKeepVendorImplV1Test.js

```diff
@@ -1,14 +1,20 @@
-import {createSnapshot, restoreSnapshot} from "./helpers/snapshot"
+const {accounts, contract, web3} = require("@openzeppelin/test-environment")
+const {createSnapshot, restoreSnapshot} = require("./helpers/snapshot")
 
 const {constants, expectRevert} = require("@openzeppelin/test-helpers")
 
-const KeepRegistry = artifacts.require("KeepRegistry")
-const BondedECDSAKeepVendorImplV1Stub = artifacts.require(
+const KeepRegistry = contract.fromArtifact("KeepRegistry")
+const BondedECDSAKeepVendorImplV1Stub = contract.fromArtifact(
   "BondedECDSAKeepVendorImplV1Stub"
 )
 
+const BN = web3.utils.BN
+const chai = require("chai")
+chai.use(require("bn-chai")(BN))
+const assert = chai.assert
```

Comment:
We are not asserting any big numbers so I think it's enough to have:
```
const chai = require("chai")
const assert = chai.assert
```

Reply:
Removed